# above(l0,l1) means l0 is above l1

string NAME=above

ObjectLocation l0
ObjectLocation l1

bool truth_value
